tika/server.go

```diff
@@ -36,20 +36,21 @@ import (
 // There is no need to create a Server for an already running Tika Server
 // since you can pass its URL directly to a Client.
 type Server struct {
-	jar  string
-	url  string // url is derived from port.
-	port string
-	cmd  *exec.Cmd
-	child *ChildOptions
+	jar       string
+	url       string // url is derived from port.
+	port      string
+	cmd       *exec.Cmd
+	child     *ChildOptions
+	javaprops map[string]string
```

Comment:
Can we export this variable and have people set it directly? Then we won't need the `AddJavaProp` function, which will be nicer when setting multiple properties.
```suggestion
	JavaProps map[string]string
```